projects/ad-manual.halex.alignment.xml

```diff
@@ -223,47 +223,47 @@ else {
                 <cf:locale country="" language="" variant="">
                     <cf:template>
                         <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-Erstellt ein GeographicalName-Objekt aus einer Lagebezeichnung.&#13;
-Eine unverschlüsselte Lagebezeichnung wird direkt als Name übernommen.&#13;
-Falls stattdessen eine verschlüsselte Lagebezeichnung vorliegt (nur in ALKIS/DLKM möglich), wird versucht den Namen über den Schlüssel in einem Index nachzuschlagen.&#13;
-Trifft keiner der beiden Fälle zu, oder kann der Name nicht nachgeschlagen werden, wird kein GeographicalName erzeugt.&#13;
-&#13;
-Die Sprache des Namens wird als Deutsch angenommen.&#13;
-&#13;
-&lt;% if (_params.nameStatus) {&#13;
-  if (_params.nameStatus == 'unpopulated') {&#13;
-    out &lt;&lt; "Der Status des Namens (`nameStatus`) wird nicht angegeben, als *nilReason* wird `other:unpopulated` angegeben."&#13;
-  }&#13;
-  else if (_params.nameStatus == 'unknown') {&#13;
-    out &lt;&lt; "Der Status des Namens (`nameStatus`) wird nicht angegeben, als *nilReason* wird `unknown` angegeben."&#13;
-  }&#13;
-  else {&#13;
-    out &lt;&lt; "Der Status des Namens (`nameStatus`) wird als ${_params.nameStatus} angegeben (siehe &lt;http://inspire.ec.europa.eu/codelist/NameStatusValue&gt;)."&#13;
-  }&#13;
-}&#13;
-else {&#13;
-  out &lt;&lt; "Der Status des Namens (`nameStatus`) wird als *official* angegeben (siehe &lt;http://inspire.ec.europa.eu/codelist/NameStatusValue&gt;)."&#13;
-}&#13;
-out &lt;&lt; "\n\n"&#13;
-if (_params.nativeness) {&#13;
-  if (_params.nativeness == 'unpopulated') {&#13;
-    out &lt;&lt; "Die Ortsüblichkeit (`nativeness`) wird nicht angegeben, als *nilReason* wird `other:unpopulated` angegeben."&#13;
-  }&#13;
-  else if (_params.nativeness == 'unknown') {&#13;
-    out &lt;&lt; "Die Ortsüblichkeit (`nativeness`) wird nicht angegeben, als *nilReason* wird `unknown` angegeben."&#13;
-  }&#13;
-  else {&#13;
-    out &lt;&lt; "Die Ortsüblichkeit (`nativeness`) wird als ${_params.nativeness} angegeben (siehe &lt;http://inspire.ec.europa.eu/codelist/NativenessValue&gt;)."&#13;
-  }&#13;
-}&#13;
-else {&#13;
-  out &lt;&lt; "Die Ortsüblichkeit (`nativeness`) wird als *endonym* angegeben (siehe &lt;http://inspire.ec.europa.eu/codelist/NativenessValue&gt;)."&#13;
-}&#13;
-out &lt;&lt; "\n\n"&#13;
-if (_params.source) {&#13;&#13;
-  out &lt;&lt; "Die Quelle des Namens (`sourceOfName`) wird als ${_params.source} angegeben."&#13;&#13;
-} else {&#13;&#13;
-  out &lt;&lt; "Die Quelle des Namens (`sourceOfName`) wird aus einer Projekt-Variable (`ADV_MODELLART`) bestimmt, falls vorhanden."&#13;&#13;
+Erstellt ein GeographicalName-Objekt aus einer Lagebezeichnung.&#xD;
```

Comment:
It would be good if you just include the relevant changes and not any additional "style" changes. Additional changes make the automated migration process more likely to run into problems trying to apply the changes.

Reply:
Ok. I have a hard time to determine though where these changes come from. Can it be related to the hale studio version in use?

Reply:
Could maybe be related to your operating system or Java version.

Reply:
The changes were only displayed to me in the compare mode - I could not make the encoding appear in any other text editor to remove it. @stempler Could you do so please?